components/threading/global_threads/thread_menu/thread_menu.tsx

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {memo, PropsWithChildren} from 'react';
+import {useIntl} from 'react-intl';
+
+import {t} from 'utils/i18n';
+
+import Menu from 'components/widgets/menu/menu';
+import MenuWrapper from 'components/widgets/menu/menu_wrapper';
+
+import Button from 'components/threading/common/button';
+import SimpleTooltip from 'components/simple_tooltip';
+
+type Props = {
+    isFollowing: boolean,
+    isSaved: boolean,
+    hasUnreads: boolean,
+    actions: {
+        follow: () => void,
+        unfollow: () => void,
+        save: () => void,
+        unsave: () => void,
+        markRead: () => void,
+        markUnread: () => void,
+        openInChannel: () => void,
+        copyLink: () => void,
+    }
+};
+
+function ThreadMenu({
+    isFollowing,
+    isSaved,
+    hasUnreads,
+    actions: {
+        follow,
+        unfollow,
+        save,
+        unsave,
+        markRead,
+        markUnread,
+        openInChannel,
+        copyLink,
+    },
+    children = (
+        <Button>
+            <i className='Icon icon-dots-vertical'/>
+        </Button>
+    ),
+}: PropsWithChildren<Props>) {
+    const {formatMessage} = useIntl();
+
+    return (
+        <MenuWrapper
+            stopPropagationOnToggle={true}
+        >
+            <SimpleTooltip
+                id='threadActionMenu'
+                content={formatMessage({
+                    id: 'threading.threadItem.menu',
+                    defaultMessage: 'Actions',
+                })}
+            >
+                {children}
+            </SimpleTooltip>
+
+            <Menu
+                ariaLabel={''}
```

Comment:
empty aria label?

Reply:
Menu requires it, waiting to to accessibility dev/work until integration stage.